bigquery/remote-function/document/document_function.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START bigquery_remote_function_document]
+import urllib.request
+
+import flask
+import functions_framework
+from google.cloud import documentai
+
+_PROJECT_ID = "YOUR_PROJECT_ID"
+_LOCATION = "us"
+_PROCESSOR_ID = "YOUR_PROCESSOR_ID"
+
+
+@functions_framework.http
+def document_ocr(request: flask.Request) -> flask.Response:
+    """BigQuery remote function to process document using OCR.
+
+    Args:
+        request: HTTP request from BigQuery
+        https://cloud.google.com/bigquery/docs/reference/standard-sql/remote-functions#input_format
+
+    Returns:
+        HTTP response to BigQuery
+        https://cloud.google.com/bigquery/docs/reference/standard-sql/remote-functions#output_format
+    """
+    try:
+        client = documentai.DocumentProcessorServiceClient()
```

Comment:
```suggestion
        client = documentai.DocumentProcessorServiceClient(
            client_options=ClientOptions(
                api_endpoint=f"{_LOCATION}-documentai.googleapis.com"
            )
        )
```

This is needed for any location outside the US